solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    let title: String
+    let description: String
+    let buttonTitle: String
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            Text(title)
+                .font(.title2)
+                .bold()
+                .frame(alignment: .leading)
+            
+            Text(description)
+                .lineLimit(3)
+
+            Button {
+                
+            } label: {
+                Text(buttonTitle)
+                    .bold()
+                    .frame(minWidth: 0, maxWidth: .infinity)
+            }.buttonStyle(SpaceButtonStyle())
+        }
+        .padding(15)
+    }
+}
+
+struct SpaceButtonStyle: ButtonStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .padding()
+            .background(.blue)
+            .foregroundColor(.white)
+            .cornerRadius(15)
+    }
+}
+
+struct LaunchDetailsBodyView_Previews: PreviewProvider {
+    static var previews: some View {
+        LaunchDetailsBodyView(title: "Details", description: "SpaceX's 20th and final Crew Resupply Mission under the original NASA CRS contract, this mission...", buttonTitle: "See more")
```

Comment:
**[SUGGESTION]** 
Consider creating also a preview to check how this would behave in Dark mode.  
```suggestion
        LaunchDetailsBodyView(model: .fixture())
            .preferredColorScheme(.light)
        LaunchDetailsBodyView(model: .fixture())
            .preferredColorScheme(.dark)
```

Also, it would be nice to have cases that represent this view with less than 3 lines and also more than 3 lines.

Reply:
"Also, it would be nice to have cases that represent this view with less than 3 lines and also more than 3 lines"

Do you mean for the preview, to have a mock with less than 3 lines and also more than 3 lines?